uint8 id
float64 heading
geometry_msgs/Vector3 position
geometry_msgs/Vector3 goal
geometry_msgs/Vector3 velocity
float64 largest_workload_piece
float64 workload